widgetry/src/widgets/panel.rs

```diff
@@ -437,6 +437,12 @@ impl Panel {
         }
     }
 
+    pub fn swap_contained_content(&mut self, ctx: &EventCtx, name: &str, widget: &mut Widget) {
```

Comment:
Not sure about this name... maybe `swap_inner_content` would be better?

Reply:
I'm a bit confused about what this is doing. Why is the first member of the row or column special and worth swapping out?
EDIT: Ah, it's because we're assuming `name` was made from a `.container()` call and ONLY has one item. Is it worth asserting that there's only one item in there below?

API-wise: maybe lock it down to `pub(crate)` unless you have use cases outside of `TabController` in mind, and add a bit of docstring?